### ISSUE TITLE
Q: crypto$Hmac "update". Method cannot be called on possibly undefined value

### ISSUE THREAD

**Issue by bigunaleksej** [facebook/flow#1273]
For example I have next method
```javascript
getHash(data): any {
    return crypto
      .createHmac('sha256', 'key')
      .update(data)
      .digest('hex');
}
```
For which flow through next error:
```
return crypto
        ^ call of method `update`
 return crypto
        ^ call of method `digest`. Method cannot be called on possibly undefined value
 return crypto
        ^ undefined
```
I think it is happened because flow has next annotation 
```
type crypto$Hmac = {
  update(data: any): void;
  digest(encoding?: string): any;
}
```
Probably ```update``` method should return ```crypto$Hmac``` instead of ```void```.

Is it a mistake of type description or I did something wrong?

**Comment by owais** [comment]
@bigunaleksej  Have you found any figured out a solution/explanation to this?

**Comment by samwgoldman** [comment]
This was fixed in #1316 and will be in the next release.
